.github/workflows/e2ePerformanceTests.yml

```
@@ -33,20 +35,33 @@ jobs:
 
       - name: Get pull request details
         id: getPullRequestDetails
-        uses: Expensify/App/.github/actions/javascript/getPullRequestDetails@main
+        uses: jczekalski/App/.github/actions/javascript/getPullRequestDetails@fix/15605-run-e2e-performance-tests-for-unmerged-PRs
```

```suggestion
uses: Expensify/App/.github/actions/javascript/getPullRequestDetails@main
```
Modified for testing purposes, revert change before merging to `main`.